src/main/java/InvitationcontrollerAndPresenter/acceptInvitationController.java

```diff
@@ -0,0 +1,21 @@
+package InvitationcontrollerAndPresenter;
+
+import useCases.InvitationAcceptOrDecline.AcceptInvitationInputBoundary;
+import useCases.InvitationAcceptOrDecline.AcceptInvitationInputModel;
+
+public class acceptInvitationController {
+
+    final AcceptInvitationInputBoundary userInput;
```

Comment:
change the name to be more in line with what the Input Boundary does (i.e. creates an Invitation)